docs/verify_data/manual_audit.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Manual Audit
+sidebar_position: 3
+---
+
+For streams that continuously produce data it would be quite hard to manually re-trigger validation for all proofs, for this use-case we can utilize the "Manual Audit" section. You can find it by navigation to it using the tab control in the Data audit section.
```

Comment:
```suggestion
For streams that continuously produce data it would be quite hard to manually re-trigger validation for all proofs, so for this use-case we can utilize the "Manual Audit" section. You can navigate to it by using the tab control in the Data audit section.
```